src/utils/constants.ts

```diff
@@ -10,3 +10,5 @@ export const ACCOUNT_NOT_SET_ERROR_MESSAGE = "cannot request without setting acc
 export const MAINNET_ESPLORA_BASE_PATH = "https://blockstream.info/api";
 export const TESTNET_ESPLORA_BASE_PATH = "https://electr-testnet.do.polkabtc.io";
 export const REGTEST_ESPLORA_BASE_PATH = "http://0.0.0.0:3002";
+
+export const INDEX_URL = process.env.INTERBTC_INDEX_SERVER_URL || "http://localhost:3007";
```

Comment:
The lib is not really a standalone program, so there should be no environment variables

Reply:
Good point. I'll make it a parameter.